### ISSUE TITLE
[draw][text] text size in draw is different when face color is specified

### ISSUE THREAD

**Issue by Palaing** [red/red#3847]
**Describe the bug**
When you make a base face! with some text in the draw facet, and display it, the text size is different according to whether a "color" facet has been speficied or not in the face.

**To reproduce**
1. with the following code:
```
	l: layout [size 200x200]
	f3: make face![
		type: 'base
		offset: 0x0
		size: 160x160
		draw: [text 0x0 "Hi there"]
	]
	append l/pane f3
	view l
```
You get this:
![image](https://user-images.githubusercontent.com/344813/56292174-fa72f480-6126-11e9-8906-93c90f4eec71.png)
2. With the following code (added `color: snow`):
```
	l: layout [size 200x200]
	f3: make face![
		type: 'base
		offset: 0x0
		size: 160x160
		color: snow
		draw: [text 0x0 "Hi there"]
	]
	append l/pane f3
	view l
```
you get this instead:
![image](https://user-images.githubusercontent.com/344813/56292523-a3215400-6127-11e9-8bcc-7fe5e69653fe.png)

**Expected behavior**
Text size should not have changed

**Screenshots**
see above

**Platform version (please complete the following information)**
```
Red 0.6.4 for Windows built 13-Apr-2019/8:27:16+02:00 commit #fb18d4e
```

**Comment by hiiamboris** [comment]
On my W7 I get the second result in both cases. Same build.

**Comment by meijeru** [comment]
On W10 I get same size text, but in second case it is ** bold**

**Comment by hiiamboris** [comment]
As there have been changes to draw and it should no longer use face's font... is this issue still relevant?

**Comment by Palaing** [comment]
Issue solved indeed with red-21feb20-3c68ef12.exe
